packages/block-library/src/gallery/block.json

```diff
@@ -6,7 +6,7 @@
 			"type": "array",
 			"default": [],
 			"source": "query",
-			"selector": ".blocks-gallery-item",
+			"selector": ".wp-block-image",
```

Comment:
Does seem to be possible to use a selector to get attribute data from InnerBlocks, but maybe there is a special syntax for that